src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonListener.cs

```diff
@@ -52,7 +50,6 @@ public SingletonListener(FunctionDescriptor method, SingletonAttribute attribute
             if (_lockHandle == null)
             {
                 string msg = string.Format(CultureInfo.InvariantCulture, "Unable to acquire Singleton lock ({0}).", _lockId);
-                _trace.Verbose(msg, source: TraceSource.Execution);
                 _logger?.LogDebug(msg);
```

Comment:
Same question here and throughout about preservation of category/source info

Reply:
Same answer :-)